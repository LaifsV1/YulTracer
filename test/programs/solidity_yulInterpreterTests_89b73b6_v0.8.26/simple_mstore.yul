{
  mstore(10, 11)

  // Added memory dump:
  printhex(mload(0x20))
}
// ----
// Trace:
// Memory dump:
//     20: 0000000000000000000b00000000000000000000000000000000000000000000
// Storage dump:
// Transient storage dump:
