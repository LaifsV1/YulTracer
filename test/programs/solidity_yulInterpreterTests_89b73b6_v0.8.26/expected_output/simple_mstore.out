0000000000000000000b00000000000000000000000000000000000000000000
